src/queue/queue-slot.ts

```diff
@@ -5,4 +5,5 @@ export interface QueueSlot {
   gameClass: Tf2ClassName;
   playerId: string | null;
   ready: boolean;
+  canMakeFriends: boolean;
```

Comment:
why not make this nullable? Then you don't need half the changes in this PR

you could then default this to false

Reply:
`canMakeFriendsWith` is an optional property now.